### PR TITLE
Deactivate add relation buttons with empty inputs.

### PR DESCRIPTION
**Pull request by manwithtwowatches**
https://www.openproject.org/work_packages/15211

### CODE REVIEW

File: public/templates/work_packages/tabs/_add_work_package_relation.html [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
-   type="text"
-   autocomplete="off">
-<div id="related_issue_candidates-{{ handler.relationsId }}"
- class="autocomplete related-issue-candidates"></div>
+<form name="add_relation_form">
+  <button class="button"
+          title="{{ btnTitle }}"
+          ng-bind-html="btnIcon + ' ' + btnTitle"
+          ng-click="handler.addRelation(this)"
+          ng-disabled="add_relation_form.$invalid">
```
Comment (hschink) [inline public/templates/work_packages/tabs/_add_work_package_relation.html:6]: I guess for the parent relation we want to allow setting it to `null`, right?

Comment (manwithtwowatches) [inline public/templates/work_packages/tabs/_add_work_package_relation.html:6]: to remove the parent? why don't we just have a trash icon like the others?

Comment (hschink) [inline public/templates/work_packages/tabs/_add_work_package_relation.html:6]: @manwithtwowatches no, there is no trash icon. So I guess we need to introduce it for parents.

Additionally, after adding a parent the button is not disabled right away. It works for the other relations but not for the parent/child one.

**Comment by hschink** [comment]
@manwithtwowatches Removing a parent works like it should. But adding a parent work package after deletion behaves strange: It just closes the parent relation area. So you may do the following to reproduce the issue:
1. Add a parent to a work package
2. Remove the parent from the work package
3. Add a parent to the work package again
4. The parent relation section is closed.

**Pull request merged**